cmd/gvproxy/main.go

```diff
@@ -334,7 +345,7 @@ func run(ctx context.Context, g *errgroup.Group, configuration *types.Configurat
 	mux.Handle("/services/forwarder/all", vn.Mux())
 	mux.Handle("/services/forwarder/expose", vn.Mux())
 	mux.Handle("/services/forwarder/unexpose", vn.Mux())
-	httpServe(ctx, g, ln, mux)
+	httpServe(ctx, g, ln, vn.Mux())
```

Comment:
I think we need to pass `mux` instead of `vn.Mux()` as 
```go
mux.Handle("/services/forwarder/all", vn.Mux())
mux.Handle("/services/forwarder/expose", vn.Mux())
mux.Handle("/services/forwarder/unexpose", vn.Mux())
```
will never be served

Reply:
Or 
```go
mux := http.NewServeMux()
mux.Handle("/services/forwarder/all", vn.Mux())
mux.Handle("/services/forwarder/expose", vn.Mux())
mux.Handle("/services/forwarder/unexpose", vn.Mux())
```
 is completely unnecessarily, as https://github.com/containers/gvisor-tap-vsock/blob/13546a0ce04f1fd3e114d929239ee63bc5c61aa4/pkg/virtualnetwork/services.go#L47 register own handler for `/services/forwarder/all` path

I think we need to refactor all http endpoints and handlers registration it separate issue, as for now endpoints/handlers are scattered across the code.